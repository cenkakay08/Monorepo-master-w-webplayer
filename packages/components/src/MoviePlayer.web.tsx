import WebPlayer from 'react-native-web-player'
import { View } from 'react-native'

export function Movieplayer() {
    return <View>
    <WebPlayer style={{ width: 800, height: 500 }} baseURL= "http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4"/>
    </View>
}