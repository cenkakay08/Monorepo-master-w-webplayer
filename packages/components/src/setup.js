import React, { Component } from 'react';
import { Provider } from 'react-redux';
import { PersistGate } from 'redux-persist/lib/integration/react';
import withTheme from './styles/withTheme';
import GlobalStyle from './styles/GlobalStyle';
import createStore from './redux/createStore';

const store = createStore();

export default function setupApp(App) {
    const ThemedApp = withTheme(App);

    class Root extends Component {
        render() {
            return (
                <Provider store={store}>
                    <PersistGate persistor={store.persistor} loading={null}>
                        <React.Fragment>
                            <ThemedApp />
                            <GlobalStyle />
                        </React.Fragment>
                    </PersistGate>
                </Provider>
            );
        }
    }

    return Root;
}
