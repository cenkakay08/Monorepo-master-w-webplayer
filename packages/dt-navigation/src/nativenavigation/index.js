import { navigate, goBack, replace } from './react-navigation-context';

const useNativeNavigation = () => {
  return {
    navigate,
    replace,
    goBack,
  };
};

export { useNativeNavigation };
