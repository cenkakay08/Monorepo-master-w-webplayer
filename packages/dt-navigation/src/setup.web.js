import React from 'react';
import { View } from 'react-native';

export default function setupNavigation(AppNavigator) {
  const Root = () => {
    return (
      <React.Fragment>
        <View>
          <AppNavigator />
        </View>
      </React.Fragment>
    );
  };

  return Root;
}
