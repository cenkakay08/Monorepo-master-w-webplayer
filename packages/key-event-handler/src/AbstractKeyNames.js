/**
 * @file Contains abstract key names for keymap.
 * @author DIGITURK\DTHSARI - HAMZA SARI (TECHNOLOGY) <hamza.sari@digiturk.com.tr>
 */

/**
 * Abstract key names.
 */
export const AbstractKeyNames = {
  LEFT: 'LEFT',
  RIGHT: 'RIGHT',
  UP: 'UP',
  DOWN: 'DOWN',
  FCT_1: 'FCT_1',
  FCT_2: 'FCT_2',
  FCT_3: 'FCT_3',
  FCT_4: 'FCT_4',
  PLAY: 'PLAY',
  PAUSE: 'PAUSE',
  PLAY_PAUSE: 'PLAY_PAUSE',
  STOP: 'STOP',
  FORWARD: 'FORWARD',
  REWIND: 'REWIND',
  PREV: 'PREV',
  NEXT: 'NEXT',
  NUM_1: 'NUM_1',
  NUM_2: 'NUM_2',
  NUM_3: 'NUM_3',
  NUM_4: 'NUM_4',
  NUM_5: 'NUM_5',
  NUM_6: 'NUM_6',
  NUM_7: 'NUM_7',
  NUM_8: 'NUM_8',
  NUM_9: 'NUM_9',
  NUM_0: 'NUM_0',
  MUTE: 'MUTE',
  VOL_UP: 'VOL_UP',
  VOL_DOWN: 'VOL_DOWN',
  PROG_UP: 'PROG_UP',
  PROG_DOWN: 'PROG_DOWN',
  INFO: 'INFO',
  HOME: 'HOME',
  SETTINGS: 'SETTINGS',
  VIDEO_SETTINGS: 'VIDEO_SETTINGS',
  AUDIO_SETTINGS: 'AUDIO_SETTINGS',
  MODECHANGE: 'MODECHANGE',
  REC_TIMER: 'REC_TIMER',
  TV_RAD: 'TV_RAD',
  PVR: 'PVR',
  REC: 'REC',
  BACK_SPACE: 'BACK_SPACE',
  BACK: 'BACK',
  OK: 'OK',
  SPECIAL: 'SPECIAL',
  MENU: 'MENU',
  POWER: 'POWER',
  GUIDE: 'GUIDE',
  EXIT: 'EXIT',
  DEL: 'DEL',
  OPTION: 'OPTION',
  PIP: 'PIP',
  VOD: 'VOD',
  HELP: 'HELP',
  AUDIO: 'AUDIO',
  SUBTITLES: 'SUBTITLES',
  FAVORITES: 'FAVORITES',
  SEARCH: 'SEARCH',
  VIEW: 'VIEW',
};
