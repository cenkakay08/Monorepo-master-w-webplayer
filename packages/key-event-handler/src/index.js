/**
 * @file Entry point for package and manages all exports from package local to outside of the package.
 * @author DIGITURK\DTHSARI - HAMZA SARI (TECHNOLOGY) <hamza.sari@digiturk.com.tr>
 */

/**
 * @import KeyBinder from './components/KeyBinder';
 * @import { keysReducer } from './redux/reducer';
 */
import KeyBinder from './components/KeyBinder';
import { keysReducer } from './redux/reducer';

export { KeyBinder, keysReducer };
