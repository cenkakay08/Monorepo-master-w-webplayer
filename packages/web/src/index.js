import { AppRegistry } from 'react-native';

import Root from 'shared/src/setup';

AppRegistry.registerComponent('myprojectname', () => Root);
AppRegistry.runApplication('myprojectname', {
  rootTag: document.getElementById('root'),
});
