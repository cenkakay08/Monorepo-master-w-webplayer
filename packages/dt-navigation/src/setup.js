import React from 'react';
import { View, Platform, KeyboardAvoidingView } from 'react-native';
import { NavigationContainer } from '@react-navigation/native';
import 'react-native-gesture-handler';
import { navigationRef } from './nativenavigation/react-navigation-context';

export default function setupNavigation(AppNavigator) {
  const Root = () => {
    return (
      <NavigationContainer ref={navigationRef}>
        <>
          <View>
            <AppNavigator />
            <KeyboardAvoidingView
              behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
            />
          </View>
        </>
      </NavigationContainer>
    );
  };

  return Root;
}
