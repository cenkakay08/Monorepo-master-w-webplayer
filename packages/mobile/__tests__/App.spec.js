/**
 * @format
 */

import 'react-native';
import React from 'react';

import {App} from 'components/src/App';

jest.mock("react-native-video", () => "Video");

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  renderer.create(<App />);
});
