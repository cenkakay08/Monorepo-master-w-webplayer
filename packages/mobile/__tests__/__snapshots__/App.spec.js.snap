// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the shared app 1`] = `
<View
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    collapsable={true}
    pointerEvents="box-none"
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <SafeAreaView>
      <ScrollView
        contentInsetAdjustmentBehavior="automatic"
        style={
          Object {
            "backgroundColor": "white",
          }
        }
      >
        <View
          accessibilityIgnoresInvertColors={true}
          style={
            Object {
              "backgroundColor": "#F3F3F3",
              "paddingBottom": 40,
              "paddingHorizontal": 32,
              "paddingTop": 96,
            }
          }
        >
          <Image
            accessibilityRole="image"
            source={
              Object {
                "testUri": "../Libraries/NewAppScreen/components/logo.png",
              }
            }
            style={
              Array [
                Object {
                  "bottom": 0,
                  "left": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                },
                Object {
                  "height": undefined,
                  "width": undefined,
                },
                Object {
                  "marginBottom": -192,
                  "marginLeft": -128,
                  "opacity": 0.2,
                  "overflow": "visible",
                  "resizeMode": "cover",
                },
              ]
            }
          />
          <Text
            style={
              Object {
                "color": "#000",
                "fontSize": 40,
                "fontWeight": "600",
                "textAlign": "center",
              }
            }
          >
            Welcome to React
          </Text>
        </View>
        <View
          style={
            Object {
              "backgroundColor": "white",
            }
          }
        >
          <View
            style={
              Object {
                "marginTop": 32,
                "paddingHorizontal": 24,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "black",
                  "fontSize": 24,
                  "fontWeight": "600",
                }
              }
            >
              Code sharing using Monorepo
            </Text>
            <Text
              style={
                Object {
                  "color": "gray",
                  "fontSize": 18,
                  "fontWeight": "400",
                  "marginTop": 8,
                }
              }
            >
              Edit
               
              <Text
                style={
                  Object {
                    "fontWeight": "700",
                  }
                }
              >
                packages/components/App.tsx
              </Text>
               
              to change this screen and then come back to see your edits (in the phone or the browser).
            </Text>
          </View>
          <View
            style={
              Object {
                "marginTop": 32,
                "paddingHorizontal": 24,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "black",
                  "fontSize": 24,
                  "fontWeight": "600",
                }
              }
            >
              Web support via react-native-web
            </Text>
            <Text
              style={
                Object {
                  "color": "gray",
                  "fontSize": 18,
                  "fontWeight": "400",
                  "marginTop": 8,
                }
              }
            >
              Run
               
              <Text
                style={
                  Object {
                    "fontWeight": "700",
                  }
                }
              >
                yarn workspace web start
              </Text>
               
              to open this app in the browser.
            </Text>
            <Text
              style={
                Object {
                  "color": "gray",
                  "fontSize": 18,
                  "fontWeight": "400",
                  "marginTop": 8,
                }
              }
            >
              It will share the same code from mobile, unless you create platform-specific files using the
               
              <Text
                style={
                  Object {
                    "fontWeight": "700",
                  }
                }
              >
                .web.tsx
              </Text>
               extension (also supports 
              <Text
                style={
                  Object {
                    "fontWeight": "700",
                  }
                }
              >
                .android
              </Text>
              ,
               
              <Text
                style={
                  Object {
                    "fontWeight": "700",
                  }
                }
              >
                .ios
              </Text>
              ,
               
              <Text
                style={
                  Object {
                    "fontWeight": "700",
                  }
                }
              >
                .native
              </Text>
              , etc).
            </Text>
          </View>
        </View>
        <View>
          <Video
            paused={false}
            resizeMode="cover"
            source={
              Object {
                "uri": "https://d23dyxeqlo5psv.cloudfront.net/big_buck_bunny.mp4",
              }
            }
            style={
              Object {
                "height": 500,
                "position": "relative",
                "width": 800,
              }
            }
          />
        </View>
      </ScrollView>
    </SafeAreaView>
  </View>
</View>
`;
